src/main/java/com/ghostipedia/cosmiccore/mixin/PlayerBreathingMixin.java

```diff
@@ -0,0 +1,57 @@
+package com.ghostipedia.cosmiccore.mixin;
+
+import fuzs.thinair.api.v1.AirQualityLevel;
+import fuzs.thinair.helper.AirQualityHelperImpl;
+import net.minecraft.tags.FluidTags;
+import net.minecraft.tags.TagKey;
+import net.minecraft.world.effect.MobEffectInstance;
+import net.minecraft.world.effect.MobEffects;
+import net.minecraft.world.entity.EquipmentSlot;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.Items;
+import net.minecraft.world.level.material.Fluid;
+import org.spongepowered.asm.mixin.*;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.Redirect;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+@Debug(export = true)
```

Comment:
Shouldn't we disable the debug export outside of dev environments? Or does this only apply for when in intellij

Reply:
i think it only stays during dev but i can remove it